src/test/DeltaNeutralLpV3.t.sol

```diff
@@ -182,11 +190,29 @@ contract DeltaNeutralV3EthTest is DeltaNeutralV3Test {
         vm.createSelectFork("ethereum", 16_394_906);
     }
 
-    function _usdc() internal pure override returns (address) {
+    function _asset() internal pure override returns (address) {
         return 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
     }
 
     function _deployStrategy() internal override {
         strategy = SslpV3.deployEth(vault);
     }
 }
+
+contract DeltaNeutralV3EthWethTest is DeltaNeutralV3Test {
+    function _selectFork() internal override {
+        vm.createSelectFork("ethereum", 16_394_906);
+    }
+
+    function _asset() internal pure override returns (address) {}
```

Comment:
I'm pretty sure the asset should be WETH here.

Reply:
Removed in new version